src/about.html

```diff
@@ -46,7 +46,7 @@ <h1 style="color:white" class="logos" >'Logo1'</h1>
         </aside>
 
         <aside class="right_aside">
-            <p style="color:white; margin: 5px;"><strong>ABOUT US</strong></p>
+            <p class="about_right_aside_text" ><strong>ABOUT US</strong></p>
```

Comment:
about_right_aside_text is not good namin, please reffer to articles about smacss or atomic